### ISSUE TITLE
README is not updated

### ISSUE THREAD

**Issue by emepetres** [MSO4SC/cloudify-im-extension#4]
The README must be updated to explain how the extension should be used.

**Comment by victorsndvg** [comment]
This is already solved in #8

You can check this branch by using next line in your blueprint:
```
  - http://raw.githubusercontent.com/MSO4SC/cloudify-im-extension/add/keypair/im.yaml
```
Note:
Before accepting the pull request we need to replace:
```
        source: https://github.com/MSO4SC/cloudify-im-extension/archive/add/keypair.zip
```
with:
```
        source: https://github.com/MSO4SC/cloudify-im-extension/archive/master.zip
```
in  the `im.yaml` file.

**Comment by victorsndvg** [comment]
please, feel free to close the issue if it's successfully fixed
